src/cupid_ai/backend/helper.py

```diff
@@ -39,16 +36,19 @@ def analyze_image_video(video_path):
     if video_file.state.name == "FAILED":
         return {"error": "Video processing failed."}
     prompt = get_prompt("./src/cupid_ai/backend/prompts/feature_extraction.txt")
-    model = genai.GenerativeModel('gemini-1.5-flash-latest')
-    response = model.generate_content([prompt, video_file], request_options={"timeout": 600})
+    model = genai.GenerativeModel("gemini-1.5-flash-latest")
+    response = model.generate_content(
+        [prompt, video_file], request_options={"timeout": 600}
+    )
     genai.delete_file(video_file.name)
-    json_response = response.text.lstrip('`json').rstrip('`')
+    json_response = response.text.lstrip("`json").rstrip("`")
     if not json_response:
         return {"error": "No response received from GenAI model."}
     data = json.loads(json_response)
-    print(data.get('HairColor'))
+    print(data.get("HairColor"))
     return data
 
+
 def get_attractiveness_score(video_path):
```

Comment:
feels like this should be in a different folder (/backend/helpers.py is pretty general, but the attractiveness_score is specific to the User or Profile module)

Reply:
move to attractivenss_score_helper.py